src/coreclr/vm/clsload.cpp

```diff
@@ -951,17 +951,6 @@ TypeHandle ClassLoader::LoadConstructedTypeThrowing(TypeKey *pKey,
         typeHnd = pKey->GetModule()->LookupFullyCanonicalInstantiation(pKey->GetTypeToken(), &existingLoadLevel);
     }
 
-    if (typeHnd.IsNull())
-    {
-        typeHnd = LookupTypeHandleForTypeKey(pKey);
```

Comment:
This stuff shouldn't be removed. Only the call to the logger should go.

Reply:
Ugh. I was stuck on this for about 10 minutes trying to determine if I should delete this branch. Thanks for confirming.